src/pkgmt/versioneer.py

```diff
@@ -178,7 +192,7 @@ def version(
         expand_github_from_changelog(path=versioner.path_to_changelog)
 
         # TODO: make it compatible with VersionerNonSetup
-        if not version_package:
+        if not version_file:
```

Comment:
let's get rid of this if statement. the line should be executed regardless of which type of project we're using